Changes

```diff
@@ -1,6 +1,8 @@
 
 8.67  2020-11-30
   - Improved Mojolicious::Commands to throw an exception for invalid commands.
+  - Fixed a bug where request_id attribute in Mojo::Message::Request would revert to previous value(s) whenever Morbo
+    restarted -- e.g., responding to file modification.  (DabeDotCom)
```

Comment:
Please don't add Changes entries in PRs.

Reply:
*Mea culpa*  I've backed it out.